database/resources/events.json

```
@@ -0,0 +1,24 @@
+[
+  {
+    "name": "Hackathon",
+    "description": "What do I need to know for a hackathon?",
+    "url": "https://tips.hackathon.com/article/what-is-a-hackathon",
+    "category": "resources",
+    "subcategory": "events"
+  },
+  {
+    "name": "Your First Hackathon: A Survival Guide",
+    "description": "How To Prepare For Your First Hackathon.",
+    "url": "https://blog.teamtreehouse.com/your-first-hackathon",
+    "category": "resources",
+    "subcategory": "events"
+  },
+  
+  {
+    "name": "The 8 Best Websites to Find a Hackathon",
+    "description": "A list of websites to Register for Hackathons.",
```

[praise] See you got it here! :) This is exactly what I meant by describing the source! :) 
```suggestion
    "description": "A list of websites to Register for Hackathons.",
```